packages/react-native-reanimated/src/layoutReanimation/defaultTransitions/FadingTransition.ts

```diff
@@ -29,7 +29,7 @@ export class FadingTransition
     const delayFunction = this.getDelayFunction();
     const callback = this.callbackV;
     const delay = this.getDelay();
-    const duration = this.durationV ?? 500;
+    const duration = (this.durationV ?? 500) / 2;
```

Comment:
What do you think about:
```suggestion
    const halfAnimationDuration = this.durationV / 2 ?? 250;
```

Reply:
The current change is consistent with implementations of both JumpingTransition and SequencedTransition:


https://github.com/software-mansion/react-native-reanimated/blob/82d1d9582d96b463ab95d6ba9b8c1c20e7aa867b/packages/react-native-reanimated/src/layoutReanimation/defaultTransitions/JumpingTransition.ts#L33

https://github.com/software-mansion/react-native-reanimated/blob/82d1d9582d96b463ab95d6ba9b8c1c20e7aa867b/packages/react-native-reanimated/src/layoutReanimation/defaultTransitions/SequencedTransition.ts#L44

Reply:
so, to keep it consistent we can change syntax in `JumpingTransition.ts` and `SequencedTransition.ts` too

Reply:
Sure, I'll update them!
However I would prefer:
`const halfDuration = (this.durationV ?? 500) /2;` instead of
`const halfDuration = this.durationV / 2 ?? 250;`
It's just easier to see that the default is 500, not 250